charts.d/ap.chart.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
> can't we remove bashism?

mysql and opensips depend heavily on it.

I am afraid it won't be easy (mainly due to arrays). In fact I prefer to re-write most BASH plugins to another language (node, python, whatever).

In all cases though, shell plugins should be an available option and the only way to achieve acceptable speeds with shell, is BASH. (I know BASH is the slowest of all shells, however its arrays make it especially userful and speedy for repeating tasks, compared to any other shell).